javascript/organizing-js/es6-modules.md

```diff
@@ -12,21 +12,21 @@ After completing this lesson, you will be able to:
 - Describe what a JavaScript module bundler like webpack is.
 - Explain what the concepts "entry" and "output" mean as relates to webpack.
 - Briefly explain what a development dependency is.
-- Explain what "transpiling code" means and how it relates to frontend development.
-- Briefly describe what a task runner is and how it's used in frontend development.
+- Explain what "transpiling code" means and how it relates to front-end development.
```

Comment:
Please refer to the feedback here for hyphenation, and please revert this :) https://github.com/TheOdinProject/curriculum/pull/22354

Reply:
I thought the consensus and the standard going forward was that adjectival use would be hyphenated and nouns would be one word? These are adjectives.

Reply:
In [this comment](https://github.com/TheOdinProject/curriculum/pull/22354#issuecomment-881530266), there is a preference to non-hyphenated, regardless. 

Additionally, Zach's comment
> There really isn't an industry standard here when we write "back end", "back-end", or "backend". **Due to this, and how the backend is discussed as a cohesive idea amongst English-speaking developers (plus it's less characters to type), it is my preference.**

[Marvin's feedback](https://github.com/TheOdinProject/curriculum/pull/22354#discussion_r655781492) also shows preference for the colloquial, which is non-hyphenated.

The core members of the team (who are free to chime in) recognize that in the industry, colloquialism trumps the "proper" grammar here, and we should remain consistent.

Reply:
But then Zach Baird said this: 

"The idea is that using "front-end" or "back-end" as an adjective to something like "languages" (ex. "back-end languages") is grammatically appropriate. When referring to the backend or frontend as nouns - ideas within full-stack web development - that we join the words together. The consensus is that this will be our standard in language going forward."

He then made the changes accordingly, and these were merged in by Marvin Gay.